backend/app/core/security_modern.py

```
@@ -0,0 +1,313 @@
+            token: JWT token
+            
+        Returns:
+            bool: 是否过期
+        """
+        try:
+            payload = ModernSecurityManager.decode_token(token, verify=False)
+            exp = payload.get("exp")
+            if not exp:
+                return True
+            return datetime.fromtimestamp(exp) < datetime.now(timezone.utc)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)